src/runtime_src/core/edge/common/aie_parser.cpp

```diff
@@ -80,6 +80,22 @@ get_hw_gen(const pt::ptree& aie_meta)
   return aie_meta.get<uint8_t>("aie_metadata.driver_config.hw_gen");
 }
 
+uint32_t
+get_partition_id(const pt::ptree& aie_meta)
+{
+  uint8_t num_col = aie_meta.get<uint8_t>("aie_metadata.driver_config.partition_num_cols");
```

Comment:
```suggestion
  auto num_col = aie_meta.get<uint8_t>("aie_metadata.driver_config.partition_num_cols");
```
Do not repeat the type.